api/FileSystemDirectoryReader.json

```diff
@@ -13,10 +13,17 @@
             "version_added": "18",
             "alternative_name": "DirectoryReader"
           },
-          "edge": {
-            "version_added": "≤18",
-            "alternative_name": "WebKitDirectoryReader"
-          },
+          "edge": [
+            {
+              "version_added": "79",
+              "alternative_name": "DirectoryReader"
+            },
+            {
+              "version_added": "14",
+              "version_removed": "79",
+              "alternative_name": "WebKitDirectoryReader"
```

Comment:
Which other browsers have had `WebKitDirectoryReader`? It seems unlikely that only Edge has ever had it.